spec/models/match_spec.rb

```diff
@@ -43,6 +43,23 @@
     end
   end
 
+  context '#winner' do
```

Comment:
additionally there are 4 unnecessary var-assignments in this context /o\

Reply:
I can't see a single unecessary assignment. No variable is only used only once or not changed after assigning

Reply:
https://github.com/turniere/turniere-backend/pull/38#pullrequestreview-240485750

Reply:
forget it I am blind

Reply:
done